phantom-scripts/rasterize_pdf.js

```diff
@@ -12,6 +12,23 @@ if (phantom.args.length < 2 || phantom.args.length > 3) {
 			format: 'A4',
 			orientation: 'portrait',
 			border: '1.5cm'
+			header: {
+            height: "1cm",
+            contents: phantom.callback(function(pageNum, numPages) {
+                return "<div><img src="http://files.parsetfss.com/54314910-905e-4901-af3c-56382e52a40e/tfss-89f75112-5812-4d65-8d51-0a947b2ecb71-st_logo.gif" style="text-align:left; display: block "/></div>";
+            })
```

Comment:
Nevermind, realized I could paste it into the URL. We can't have a specific customer logo in all reports. Should we accept this HTML as a URL parameter and embed it if supplied?